pilz_msgs/srv/IsBrakeTestRequired.srv

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2019 Pilz GmbH & Co. KG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+---
+int8 result
+
+int8 NOT_REQUIRED = -1
```

Comment:
You could even abstract this by creating a Messge with just the result. See for example `moveit_msg/MoveItErrorCodes.msg` with usage in different Action/Service contexts for example in `moveit_msg/MotionPlanResponse.msg`